cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -0,0 +1,2515 @@
+//! This module defines aarch64-specific machine instruction types.
+
+#![allow(non_snake_case)]
+#![allow(unused_imports)]
+#![allow(non_camel_case_types)]
```

Comment:
Several files have `allow(non_snake_case)` and `allow(non_camel_case_types)`. Could you either remove these and change the names to follow the regular Rust conventions, or add comments explaining why exceptions to the rules are justified in these files?

Reply:
Removed! None of the names were actually unconventional; I think this was left over from an earlier prototype.